WcaOnRails/app/views/users/edit.html.erb

```diff
@@ -2,185 +2,212 @@
-          <%= f.input :wca_id, as: :wca_id, disabled: !editable_fields.include?(:wca_id) %>
-        </div>
+        <% if current_user.can_edit_users? %>
+          <%= f.input :delegate_status, collection: User.delegate_statuses.keys, label_method: lambda { |k| k.titleize }, disabled: !editable_fields.include?(:senior_delegate_id) %>
+          <%= f.association :senior_delegate, disabled: !editable_fields.include?(:senior_delegate_id) %>
+          <%= f.input :region, disabled: !editable_fields.include?(:region) %>
+        <% end %>
+
+        <% unless @user.current_teams.empty? %>
+          <div class="form-group">
+            <label>Member of</label>
+            <% @user.current_teams.each do |team| %>
+              <%= link_to team.name, edit_team_path(team), class: "btn btn-xs btn-info #{"disabled" unless current_user.can_edit_team?(team)}" %>
+            <% end %>
+          </div>
+        <% end %>
+
+        <%= f.submit 'Save', class: "btn btn-primary" %>
+      <% end %>
+    </div>
+
+    <% if @user == current_user %>
+      <div class="tab-pane active" id="email">
+        <%= simple_form_for @user do |f| %>
+          <%= hidden_field_tag "section", "email" %>
+          <%= f.input :email, disabled: !editable_fields.include?(:email), hint: "Changing your email will require confirming the new email." %>
+
+          <%= f.input :current_password %>
+
+          <%= f.submit 'Save', class: "btn btn-primary" %>
+        <% end %>
       </div>
 
-      <%= f.input :delegate_status, collection: User.delegate_statuses.keys, label_method: lambda { |k| k.titleize }, disabled: !editable_fields.include?(:senior_delegate_id) %>
-      <%= f.association :senior_delegate, disabled: !editable_fields.include?(:senior_delegate_id) %>
-      <%= f.input :region, disabled: !editable_fields.include?(:region) %>
-    <% end %>
+      <div class="tab-pane active" id="preferences">
+        <%= simple_form_for @user do |f| %>
+          <%= hidden_field_tag "section", "preferences" %>
+          <%= f.input :preferred_event_ids, as: :events_picker, allowed_events: Event.all_official, selected_events: @user.preferred_events,
+                                            disabled: !editable_fields.include?(:preferred_event_ids), hint: "Selecting your preferred events makes registration for competitions quicker." %>
 
-    <% unless @user.current_teams.empty? %>
-      <div class="form-group">
-        <label>Member of</label>
-        <% @user.current_teams.each do |team| %>
-          <%= link_to team.name, edit_team_path(team), class: "btn btn-xs btn-info #{"disabled" unless current_user.can_edit_team?(team)}" %>
+          <%= f.submit 'Save', class: "btn btn-primary" %>
+        <% end %>
+      </div>
+
+      <div class="tab-pane active" id="password">
+        <%= simple_form_for @user do |f| %>
+          <%= hidden_field_tag "section", "password" %>
+          <%= f.input :password, :autocomplete => "off", class: "form-control" %>
```

Comment:
nit: whenver possible, we don't use the => syntax, change this to `autocomplete: "off"`